penguin/web/routes.py

```
@@ -4759,9 +4846,24 @@ async def get_task(task_id: str, core: PenguinCore = Depends(get_core)):
 # @router.put("/api/v1/tasks/{task_id}")
 # async def update_task(...):
 
-# Temporarily disabled - delete_task method not implemented in ProjectManager
-# @router.delete("/api/v1/tasks/{task_id}")
-# async def delete_task(...):
+@router.delete("/api/v1/tasks/{task_id}")
+async def delete_task(task_id: str, core: PenguinCore = Depends(get_core)):
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)